sktime/transformers/resizing.py

```diff
@@ -0,0 +1,72 @@
+import numpy as np
+import pandas as pd
+from scipy import interpolate
+
+from sktime.transformers.base import BaseTransformer

```

Comment:
Please inherit from `BaseSeriesToSeriesTransformer`

Reply:
May be, BaseSingleSeriesTransformer from sktime.transformers.single_series.base?
Cant' find BaseSeriesToSeriesTransformer.

Reply:
ah sorry, it's called [`BaseSeriesAsFeaturesTransformer`](https://github.com/alan-turing-institute/sktime/blob/dev/sktime/transformers/series_as_features/base.py)